coala_quickstart/generation/FileGlobs.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from coala_quickstart.generation.Utilities import comma_split
+from coala_quickstart.interaction.Question import ask_question
+from coala_quickstart.Strings import glob_help
+from coalib.collecting.Collectors import collect_files
+
+
+def get_file_glob_list(printer, prompt, default="**"):
+    """
+    Get a list of glob expressions to collect files from.
+
+    :param printer:
+        A ``ConsolePrinter`` object.
+    :param prompt:
+        Question to present to the user.
+    :param default:
+        Default value; ``**`` will match every file.
+    :return:
+        The list of glob expressions.
+    """
+    response = ask_question(
+        printer,
+        prompt,
+        default=default)
+
+    return comma_split(response)
+
+
+def get_project_files(log_printer, printer, project_dir):
+    """
+    Gets the list of files matching files in the user's project directory
+    after prompting for glob expressions
+
+    :param log_printer:
+        A ``LogPrinter`` object.
+    :param printer:
+        A ``ConsolePrinter`` object.
+    :return:
+        A list file paths matching the files.
+    """
+    printer.print(glob_help)
+    file_globs = get_file_glob_list(
+        printer,
+        "Which files do you want coala to run on?")
```

Comment:
As a comment or do you mean to the user in console? Because the question will be followed by `[**]` on console